en/render/camera.md

```diff
@@ -1,39 +1,133 @@
-- `zoomRatio`: Specifies the zoom ratio of the camera, the larger the value, the larger the image.
+- cullingMask
 
-The Camera component will move along with the node it is attached to, and we can imagine the node is a hand holding the camera, which will only shoot his targets, and the FOV of the camera is the device screen size.
+`cullingMask` will determine which parts of the scene the camera is used to render. The `cullingMask` in the camera component in the **Properties** lists the mask options currently available, and you can combine them to generate `cullingMask` by selecting these options.
 
-## Example
+For example, the `cullingMask` setting in the following figure indicates that the camera is used only to render the UI part of the game, that the UI part of the game does not need to be moved, and that the game node may move out of the screen, requiring another camera to follow the game node.
 
-We use a scene instance to explain how the Camera component are used.
+![camera-1](./camera/camera-1.png)
 
-Suppose we are making a physics game: we have physics colliders and tiled map as the background, we have `hero` node as player character. Our camera needs to follow the hero's movement so we can see him travel across the scene.
+You can add or change groupings through **Group Manager** in the **Project Settings**, which is the corresponding mask.
 
-![Camera-1](./camera/camera-1.png)
+- zoomRatio
 
-Here we need to create a new empty node and rename it to camera, use this node as a camera, using a single node as a camera node will be more flexible. Of course, we can also directly add the Camera component to the hero node, but this camera can only at the exact position of hero, and can not implement any smooth follow effect. 
+Specifies the zoom ratio of the camera, and the larger the value, the larger the image displayed.
 
-Select the camera node in the **Node Tree** and click on the `Add Component -> Add Other Component -> Camera` button below the **Properties** to add the Camera component to the camera node.
+- clearFlags
 
-<img src="./camera/camera-2.png" style="width:50%;height:50%"></img>
+Specifies the cleanup action that needs to be made when rendering the camera.
 
-Here the Camera component adds three nodes to the `targets` property, that is, we need the camera to shoot these three nodes. And we also added a `camera-control` custom components, the role of this component is mainly to move the camera node to follow the hero node.
+![camera-2](./camera/camera-2.png)
 
-This example can be found in the `tiled` example in the [Physics example](https://github.com/2youyou2/physics-example) project.
+- backgroundColor
 
-You can also refer to the [Camera Demo](https://github.com/cocos-creator/demo-camera), contains examples of the usage of camera.
+When you specify that the camera needs to clear the color, the camera uses the set background color to clear the scene.
```

Comment:
When the camera needs to clear the color， the camera will use the background color to clear the scene。